src/components/Radio/styles.ts

```diff
@@ -48,16 +48,21 @@ export const Check = styled.span`
 type StyledLabelProps = {
   checked: boolean;
   disabled: boolean;
+  hasError: boolean;
 };
 
 export const StyledLabel = styled.label<StyledLabelProps>`
   width: 100%;
   position: relative;
   border-radius: ${({ theme }) => theme.utilities.borderRadius};
-  border: ${({ theme, checked }) =>
-    `solid ${theme.utilities.inputBorderWidth} ${
-      checked && !theme.utilities.useDefaultFocusRect ? theme.colors.primary : theme.colors.inputBorder
-    }`};
+
```

Comment:
possibly split out thickness and colour, have key as string and value as string function